src/main/java/org/spongepowered/common/config/category/WorldCategory.java

```diff
@@ -208,6 +209,15 @@ public void setMobSpawnRange(int range) {
         return this.portalAgents;
     }
 
+    public String getPortalDestination(String dimensionName) {
+        // TODO I would like to know how to retrieve the default world name from here, in a better way than below
```

Comment:
I don't think there is any better way to get the default world name - it's configurable on its own. The server uses the `level-name` value in `server.properties`, and the client uses the name of the world folder.

Reply:
Is this safe to use this method here ?